package.json

```diff
@@ -19,8 +19,8 @@
   },
   "scripts": {
     "watch": "tsc -w",
-    "build": "rm -rf dist; tsc -d",
-    "prepublish": "npm run build; npm test",
+    "build": "rimraf dist; tsc -d",
+    "prepublishOnly": "npm run build; npm test",
```

Comment:
Agreed. What do you think about using something like `np` for release orchestration?